package utility

import "log"

func HandleError(err error) {
	log.Println(err)
}